qucs-doc/tutorial/linblocks/content.tex

```diff
@@ -0,0 +1,215 @@
+%        File: ampli_qucs_tut.tex
+%     Created: Fri Jan 24 05:00 PM 2014 M
+% Last Change: Fri Jan 24 05:00 PM 2014 M
+% Submitted:   Sun Feb 3 08:00 PM 2019 CET
+%
+% \documentclass[a4paper,12pt]{article}
+% \usepackage{graphicx}
+% \usepackage[margin=2cm]{geometry}
+%% my parameters for formatting
+% \parindent=0pt
+% \parskip=6pt plus 6pt minus 3pt\relax
+%% my macros
+\makeatletter
+\def\fname#1{%
+  \edef\@tempa{#1}%
+   \texttt{\expandafter\strip@prefix\meaning\@tempa}}%
+\makeatother
+%
+\newcommand{\button}[1]{``#1''}
+\newcommand{\component}[1]{``#1''}
+\newcommand{\tab}[1]{\textsl{#1}}
+\newcommand{\menu}[1]{\texttt{#1}}
+\newcommand{\dropdown}[1]{\texttt{#1}}
+\newcommand{\key}[1]{\texttt{<#1>}}
+\newcommand{\var}[1]{\textsf{#1}}
+
+\newcommand{\screenshot}[1]{\par\smallskip
+  \includegraphics[width=0.7\textwidth]{#1}\par\smallskip}
+\newcommand{\twoscreenshot}[2]{\par\smallskip
+  \includegraphics[width=0.475\textwidth]{#1}\quad
+  \includegraphics[width=0.475\textwidth]{#2}\par\smallskip}
+
+% \begin{document}
+% \title{A gentle introduction to QUCS:\\ simulating a linear circuit}
+% \author{Romano Giannetti, \fname{romano.giannetti@gmail.com}}
+% \maketitle
+
+\tutsection{Presenting the Problem}
+
+This is a step by step tutorial on how to use QUCS to simulate multi-stage amplifier (with very simple and basic approach).
```

Comment:
should it be QUCS or Qucs?
In `qucs-doc` *QUCS* seems to be used about 30 times while *Qucs* about 800. We should likely use just one or the other.

Reply:
Giving that there where more things to fix, I did that too. Thanks!